app/javascript/components/aoi-card/component.jsx

```diff
@@ -128,6 +130,7 @@ class AoICard extends PureComponent {
 
     const isSubscribed = deforestationAlerts || fireAlerts || monthlySummary;
     const subscribedToAll = deforestationAlerts && fireAlerts && monthlySummary;
+    const isPending = status === 'pending';
```

Comment:
We should also stop the fetch from being triggered on mount.

Reply:
Good idea, done 👍